ecommerce_api_client/auth.py

```diff
@@ -1,11 +1,16 @@
+import datetime
+
 import jwt
+
 from requests.auth import AuthBase
 
 
 class JwtAuth(AuthBase):
     """Attaches JWT Authentication to the given Request object."""
 
-    def __init__(self, username, full_name, email, signing_key, tracking_context=None):
+    def __init__(self, username, full_name, email, signing_key, tracking_context=None, issuer=None, expires_in=30):
+        self.issuer = issuer
+        self.expires_in = expires_in
```

Comment:
For improved readability, I'd order these new variables after `signing_key` but before `tracking_context`. `tracking_context` is not part of the JWT spec, but the signing key, issuer, and expiration time are, so they should be grouped together when possible.